aws/resource_aws_eks_cluster.go

```diff
@@ -145,6 +145,14 @@ func resourceAwsEksCluster() *schema.Resource {
 							MinItems: 1,
 							Elem:     &schema.Schema{Type: schema.TypeString},
 						},
+						"public_access_cidrs": {
+							Type:     schema.TypeSet,
+							Optional: true,
+							Elem: &schema.Schema{
+								Type:         schema.TypeString,
+								ValidateFunc: validateCIDRNetworkAddress,
+							},
+						},
```

Comment:
This schema addition needs to also be done in `aws/data_source_aws_eks_cluster.go` to prevent errors in the data source -- at some point we will discuss how to address data source testing better via https://github.com/terraform-providers/terraform-provider-aws/issues/8223

```go
// in aws/data_source_aws_eks_cluster.go
"public_access_cidrs": {
	Type:     schema.TypeSet,
	Computed: true,
	Elem:     &schema.Schema{Type: schema.TypeString},
},
```